.github/workflows/comment_fr.yml

```diff
@@ -0,0 +1,37 @@
+name: Add Comment to Feature Issues
```

Comment:
For now let's remove this, not sure I want to drop a bot comment on a FR at the moment.